examples/discord/src/workflows/humanLoopDiscord.ts

```diff
@@ -0,0 +1,48 @@
+import { log, step, condition } from "@restackio/ai/workflow";
+import * as functions from "../functions";
+
+interface Input {
+  channelId: string,
+  botId: string
+}
+
+export async function humanLoopDiscordWorkflow(input: Input) {
+  let endWorkflow = false;
+  const botTag = `<@${input.botId}>`;
+
+  
+  const promptResponse = await step<typeof functions>({}).postMessageToChannel({
+    messageText: `Message this channel and tag ${botTag} for backwards messages. Message "${botTag} STOP" to end the loop`,
+    channelId: input.channelId,
+  });
+  let previousMessageId = promptResponse.id;
+
+
+  while (!endWorkflow) {
+    await new Promise(f => setTimeout(f, 1000));
```

Comment:
you an use the "sleep" function from `@restackio/ai/workflow`

Reply:
I have moved the sleep to the `scheduleWorkflow.ts` file, but I can't use the `sleep` function in that file since it's not allowed to be used outside of a workflow execution